Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationDataSource.java

```diff
@@ -52,7 +57,30 @@
      * @param dataSourceObjectId The object ID for the datasource
      */
     public CorrelationDataSource(CorrelationCase correlationCase, String deviceId, String name, long dataSourceObjectId) {
-        this(correlationCase.getID(), -1, deviceId, name, dataSourceObjectId);
+        this(correlationCase.getID(), -1, deviceId, name, dataSourceObjectId, null, null, null);
```

Comment:
This was used by the test class. However, since the test class is the only use of it, I think it's a fair point that the test class should be updated to use what we use elsewhere. And so I have done so.